pyopenms_viz/_plotly/core.py

```
@@ -2,7 +2,7 @@
 
 from abc import ABC
 
-from typing import List, Tuple, Union
+from typing import List, Tuple, Union, Any, Dict, Optional
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import

(F401)